spec/features/venues/view_venue_details_spec.rb

```diff
@@ -21,4 +21,20 @@
     expect(page).to have_content(venue.address)
     expect(page).to have_content(venue.capacity)
   end
+
+  scenario 'If T accessible, user should see message: T is nearby' do
+    venue = FactoryGirl.create(:venue, t_accessible: true)
+
+    visit venue_path(venue)
+
+    expect(page).to have_content(venue.display_t_accessible)
```

Comment:
Check the logic here. This isn't actually testing if you see the message, this is testing that whatever logic display_t_accessible returns is shown. What if your logic within that method is wrong? Or changes down the road? This should say it expects the string